.github/workflows/test.migration.yaml

```diff
@@ -0,0 +1,127 @@
+name: "Test - Migration"
+on:
+  pull_request:
+    branches: [rc, main]
+  push:
+    branches: [rc, main]
+  workflow_dispatch:
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      changed: ${{ steps.filter.outputs.changed }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Diff Changes
+        uses: dorny/paths-filter@v3
+        id: filter
+        with:
+          filters: |
+            changed:
+              - "client/**"
+              - "freighter/py/**"
+              - "synnax/**"
+              - "cesium/**"
+              - "aspen/**"
+              - "x/**"
+              - "freighter/**"
+              - ".github/workflows/test.client.yaml"
+
+  test-on-both-servers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      # Test with Main Server
+      - name: Start Main Server
+        run: docker run -d --name synnax-main -p 9090:9090 -v /data:/data -e SYNNAX_LISTEN=0.0.0.0:9090 -e SYNNAX_VERBOSE=true -e SYNNAX_INSECURE=true -e SYNNAX_MEM=true -e SYNNAX_LICENSE_KEY=${{ secrets.SYNNAX_LICENSE_KEY }} ghcr.io/synnaxlabs/synnax:latest
```

Comment:
This server is running in memory, so it does not persist data. Also the mount path you have does nothing.